applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_apply_initial_uniform_stress_field.cpp

```diff
@@ -0,0 +1,133 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_constitutive/incremental_linear_elastic_law.h"
+#include "custom_constitutive/plane_strain.h"
+#include "custom_constitutive/three_dimensional.h"
+#include "custom_processes/apply_initial_uniform_stress_field.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(ApplyInitialUniformStressFieldProcessAppliesStressesToPlaneStrainElementsInModelParts,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Note that when creating the process using python, these parameters
+    // should also contain the "model_part_name" field.
+    Parameters parameters(R"({
+        "value": [1.0, 2.0, 3.0, 4.0]
+    })");
+
+    Model model;
+    auto& rModelPart = ModelSetupUtilities::CreateModelPartWithASingle2D3NElement(model);
```

Comment:
If we slightly modify this name, we'll be following the Kratos Style Guide:
```suggestion
    auto& r_model_part = ModelSetupUtilities::CreateModelPartWithASingle2D3NElement(model);
```

This suggestion also applies to the other test cases in this file.

Reply:
Done